packages/orm/src/EntityManager.ts

```diff
@@ -1200,6 +1200,7 @@ export class EntityManager<C = unknown, Entity extends EntityW = EntityW> {
 
     try {
       const allFlushedEntities: Set<Entity> = new Set();
+      const allFlushedJoinRowsResetFns: Set<() => void> = new Set();
```

Comment:
I think we can avoid this `new Set` and use:

```
      let joinRowTodos = combineJoinRows(this.#joinRows);
```

To loop over, later in the file.